source/documentation/appendix-docs/conda.md

```diff
@@ -139,11 +114,24 @@ environment.
 conda env update -f environment.yml --prune
 ```
 
-[packrat]: https://rstudio.github.io/packrat/
-[anaconda search]: https://anaconda.org/search
-[conda cheat sheet]:
-  https://conda.io/projects/conda/en/latest/user-guide/cheatsheet.html
 
-```{r global_options, include=FALSE}
-{knitr::opts_chunk$set(eval = FALSE)}
+## Shiny
+To use conda in shiny applications, you will need to use a different dockerfile to deploy the app. The [conda branch of the rshiny-template](https://github.com/moj-analytical-services/rshiny-template/tree/conda) has an appropriate dockerfile for this purpose. This is also necessary if you wish to use Python in your Shiny application, including using `dbtools` for accessing Athena databases.
+
+## Platform limitations
+
+There are a number of limitations and pitfalls to conda management to be aware of.
+
+### R package  versions on conda
+
+While Anaconda hosts most R packages on CRAN, some R packages on Anaconda only have binaries built for certain versions, and don't have a version for `R 3.5.1` as is currently used on the Analytical Platform. You can identify the available versions by inspecting the first few characters of the Build part of the filename - for example, **r351**h96ca727.
+
+If there isn't an appropriate build for a package, attempting to `conda install` that package will result in conda attempting to match the environment to the superior (or inferior) version of R, asking if you want to install/upgrade/downgrade a long list of packages in the process. Instead, you should install the package locally via `install.packages()` or `remotes::install_github()`. For Shiny apps, you can add an `install.packages()` step to the Dockerfile to install additional packages not covered by the conda environment.yml, like so:
+```bash
+RUN R -e "install.packages('waffle', repos = 'https://cinc.rud.is')"
 ```
+
+### Environments
+- Usually when using Conda, it makes sense to have one environment per project,
```

Comment:
this section should either be included in the #environment-management section, or removed

Reply:
Shifted